terraform/production/main.tf

```diff
@@ -34,7 +34,7 @@ module "networking" {
 }
 
 module "bastion" {
-  source = "git::https://github.com/Softwire/terraform-bastion-host-aws?ref=defd0b730d75c1b64cc1e1c76cdd5dc442d6fde6"
+  source = "git::https://github.com/Softwire/terraform-bastion-host-aws?ref=0fd25245bea740e71b05293de89705dfc62f7db6"
```

Comment:
Small change to use `aws_security_group_rule` resources, so we can add in extra rules ourselves.